apollo-router/src/uplink/mod.rs

```diff
@@ -160,13 +162,40 @@ impl UplinkConfig {
 /// Regularly fetch from Uplink
 /// If urls are supplied then they will be called round robin
 pub(crate) fn stream_from_uplink<Query, Response>(
-    mut uplink_config: UplinkConfig,
+    uplink_config: UplinkConfig,
 ) -> impl Stream<Item = Result<Response, Error>>
 where
     Query: graphql_client::GraphQLQuery,
     <Query as graphql_client::GraphQLQuery>::ResponseData: Into<UplinkResponse<Response>> + Send,
     <Query as graphql_client::GraphQLQuery>::Variables: From<UplinkRequest> + Send + Sync,
     Response: Send + 'static + Debug,
+{
+    stream_from_uplink_transforming_new_response::<Query, Response, Response>(
+        uplink_config,
+        |response| Box::new(Box::pin(async { Ok(response) })),
```

Comment:
```suggestion
        |response| Box::pin(async { Ok(response) }),
```

Does it still work ?